Code/Mantid/Testing/SystemTests/tests/analysis/LoadLotsOfFiles.py

```diff
@@ -36,6 +36,9 @@
                 'MaskSANS2DReductionGUI.txt',
                 'MaskSANS2DReductionGUI_MaskFiles.txt',
                 'MaskSANS2DReductionGUI_LimitEventsTime.txt',
+				'MASK_SANS2D_FRONT_Edges_16Mar2015.xml',
+				'MASK_SANS2D_REAR_Bottom_3_tubes_16May2014.xml',
+				'MASK_SANS2D_REAR_Edges_16Mar2015.xml',
```

Comment:
please use spaces instead of tabs, i.e. in this file replace spaces with tabs and in other .py and .h and .cpp files